games-util/springlobby/Manifest

```diff
@@ -0,0 +1 @@
+DIST springlobby-0.264.tar.bz2 2626180 BLAKE2B 01ff51703c3c10c1aaeb342d2644012a6037b1271ba7e36ab7663cde7646d1c9c6bd823fe65c91a8a9e754bbcf1000ce486eeab2d1641c64fad08773c491f9a6 SHA512 f8024a98be7d6ea755c4380494415813084aadb6dcf72ed0aaeba0ddf9fbaf992ccd5455713c807c0d6feb7db606068be62d4bb2cfcb4697f0406a2ec4f404eb
```

Comment:
Please include a commit message in the commit message ;-).

Reply:
I added a basic message, is there basic information you need to see ?

Reply:
I'd like to see the words 'new package' on the summary line. The body is unnecessary if it doesn't add anything to the summary.

Reply:
Done

Reply:
Please look at other commit messages, and use the same format. In particular, package name goes first.

Reply:
Done: "games-util/springlobby: new package"